cli/command/cli.go

```diff
@@ -27,26 +27,28 @@ import (
 
 // Streams is an interface which exposes the standard input and output streams
 type Streams interface {
-	In() *InStream
-	Out() *OutStream
+	In() InputStreamer
+	Out() OutputStreamer
 	Err() io.Writer
 }
 
 // Cli represents the docker command line client.
 type Cli interface {
 	Client() client.APIClient
-	Out() *OutStream
+	Out() OutputStreamer
```

Comment:
Ah, [you're right](https://github.com/golang/go/issues/10184#issuecomment-84465873). I guess I was mistaken.

A function like this (with a better name) might still be ok in `in.go`, another option would be to create a setter for `isTerminal`.